src/main/java/uk/gov/companieshouse/accounts/association/service/CompanyService.java

```diff
@@ -32,23 +34,24 @@ public CompanyService(CompanyProfileEndpoint companyProfileEndpoint) {
 
     public Supplier<CompanyDetails> createFetchCompanyProfileRequest( final String companyNumber ) {
         final var request = companyProfileEndpoint.createFetchCompanyProfileRequest( companyNumber );
+        final var xRequestId = getXRequestId();
         return () -> {
             try {
-                LOG.debug( String.format( "Attempting to fetch company profile for company %s" , companyNumber ) );
+                LOG.debugContext( xRequestId, String.format( "Sending request to company-profile-api: GET /company/{company_number}/company-detail. Attempting to retrieve company profile for company: %s" , companyNumber ), null );
```

Comment:
same as above

Reply:
Thank you - I will remove the redundant logging statements from all of the endpoints